### PR TITLE
QuestItem Button Update

### PR DESCRIPTION
**Pull request by TULOA**
Added code to hard code quests to Nx.QItems with the corresponding ItemID  as the value since Blizzard removed the function that got the info before.

IrcDirk: Please note I did not see the update you made so some of them changes are not present in this one. I also dont know but dont think it should break anything.

### CODE REVIEW

**Comment by IrcDirk** [comment]
Sorry but i cant add those changes. Ill add comments to files.

File: Carbonite.Gathermate2_Data/Carbonite.Gathermate2_Data.toc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## Interface: 11304
```
Comment (IrcDirk) [inline Carbonite.Gathermate2_Data/Carbonite.Gathermate2_Data.toc:1]: U did not used latest Alpha version for your changes. U are using older version of Carbonite Classic. Reverting my previous changes.

File: Carbonite.Quests/Data/Quests1-10.lua [hunk 2]
```diff
@@ -1,4 +1,4 @@
-
```
Comment (IrcDirk) [inline Carbonite.Quests/Data/Quests1-10.lua:1]: Please add only changes for QuestItems update. Those quest changes are not needed.

File: Carbonite.Quests/NxQuest.lua [hunk 3]
```diff
@@ -51,6 +51,296 @@ BINDING_NAME_NxWATCHUSEITEM	= L["NxWATCHUSEITEM"]
 
 local IsClassic = select(4, GetBuildInfo()) < 20000
 
+
```
Comment (IrcDirk) [inline Carbonite.Quests/NxQuest.lua:54]: Leave Nx.List where it was, thats in Nx.Ui file, this is used in other modules so it shouldnt be moved, becouse if someone doesnt use Quest Module this change ill broke Carbonite.

File: Carbonite.Quests/NxQuest.lua [hunk 4]
```diff
@@ -11416,7 +11714,7 @@ function Nx.Quest:OnPartyMsg (plName, msg)
 
 --	Nx.prt ("OnPartyMsg %s: %s", plName, msg)
 
-	local pq = self.PartyQ or {}
```
Comment (IrcDirk) [inline Carbonite.Quests/NxQuest.lua:11419]: U are using older version of Carbonite Classic. Reverting my previous changes.

File: Carbonite.Social/Carbonite.Social.toc [hunk 5]
```diff
@@ -1,4 +1,4 @@
-## Interface: 11304
```
Comment (IrcDirk) [inline Carbonite.Social/Carbonite.Social.toc:1]: U are using older version of Carbonite Classic. Reverting my previous changes.

File: Carbonite/Carbonite.lua [hunk 6]
```diff
@@ -840,26 +840,6 @@ function Nx:SetupEverything()
 	
 	Nx.Initialized = true
 	Nx:OnPlayer_login("PLAYER_LOGIN")
-
```
Comment (IrcDirk) [inline Carbonite/Carbonite.lua:843]: U are using older version of Carbonite Classic. Reverting my previous changes.

File: Carbonite/Carbonite.toc [hunk 7]
```diff
@@ -1,4 +1,4 @@
-## Interface: 11304
```
Comment (IrcDirk) [inline Carbonite/Carbonite.toc:1]: U are using older version of Carbonite Classic. Reverting my previous changes.

File: Carbonite/NxCom.lua [hunk 8]
```diff
@@ -531,7 +531,7 @@ function Nx.Com:OnChat_msg_addon (args, distribution, target)
 	local name = target
 	if not Nx.strpos(name, "-") then
 		local realmname = GetRealmName()
-		name = name .. (realmname and "-" .. realmname or "")
```
Comment (IrcDirk) [inline Carbonite/NxCom.lua:534]: U are using older version of Carbonite Classic. Reverting my previous changes.

File: Carbonite.Notes/Carbonite.Notes.toc [hunk 9]
```diff
@@ -1,4 +1,4 @@
-## Interface: 11304
```
Comment (IrcDirk) [inline Carbonite.Notes/Carbonite.Notes.toc:1]: U are using older version of Carbonite Classic. Reverting my previous changes.

File: Carbonite.Quests/Carbonite.Quests.toc [hunk 10]
```diff
@@ -1,4 +1,4 @@
-## Interface: 11304
```
Comment (IrcDirk) [inline Carbonite.Quests/Carbonite.Quests.toc:1]: U are using older version of Carbonite Classic. Reverting my previous changes.

**Comment by TULOA** [comment]
Closing. I can update the newest but im not leaving broken quests or incorrect database entries for them. I will just leave you with the changes and use the rest myself.

**Comment by IrcDirk** [comment]
Im only maintainer at this moment and i all help is appreciated. If u want to add some changes to Carbonite u are most welcome to do this. But separate them. 

So if u are making QuestItems to work, make Pull Request for only this changes. 
If u are updating Quest Database make separate pull request for them.
And so on...

**Pull request closed without merging**